internal/play/play_test.go

```diff
@@ -33,9 +33,9 @@ func TestPlay(t *testing.T) {
 
 	api.On("GetPlayback").Return(playback1, nil).Once()
 	api.On("GetPlayback").Return(playback2, nil).Once()
-	api.On("Play", "","", []string(nil)).Return(nil)
+	api.On("Play", "","","", []string(nil)).Return(nil)
```

Comment:
Please run `go fmt` before you submit! If you're using GoLand, you can set up a file watcher to run it every time you save.
```suggestion
	api.On("Play", "", "", "", []string(nil)).Return(nil)
```